### PR TITLE
Pre release branch for CENNZnet API to release 1.3.4 version

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Pre release branch for CENNZnet API to release 1.3.4 version.

### CODE REVIEW

**Review by holygits — COMMENTED** [review]
does @cennznet/utils import need to be bumped anywhere? apart from that lgtm

**Comment by KarishmaBothara** [comment]
> does @cennznet/utils import need to be bumped anywhere? apart from that lgtm

no.. @cennznet/utils is not a dependency in any package..

File: packages/api/test/e2e/queries.e2e.ts [hunk 1]
```diff
@@ -116,7 +116,8 @@ describe('e2e queries', () => {
   describe('Staking account derived query', () => {
     it('Gets staking account details', async done => {
       const stashId = aliceStash.address;
-      const stakingAccount = await api.derive.staking.accountInfo(stashId);
+      const currentEra = await api.query.staking.currentEra();
```
Comment (holygits) [inline packages/api/test/e2e/queries.e2e.ts:119]: should be activeEra can be null only for a few blocks at genesis until era 0 starts
```suggestion
      const activeEra = await api.query.staking.activeEra().unwrap().index;
```

**Review by holygits — APPROVED** [review]

**Comment by holygits** [comment]
need to regen docs for wiki

**Pull request merged**